frontend/pages/home.tsx

```diff
@@ -0,0 +1,59 @@
+import { ReactElement } from 'react';
+import Card from '../components/Card';
+import styled from 'styled-components';
+import Image from 'next/image';
+import homepagePic from '../public/home-image.png';
+
+const HomeContainer = styled.div`
+  background-color: #ef8b6f;
+  display: flex;
+  flex-direction: column;
+  height: 100vh;
+  padding: 2rem;
+`;
+
+const HomeTitle = styled.h1`
```

Comment:
Small discrepancy with the design prototype. In the prototype, it doesn't seem to be bold

Reply:
I think h1 inherits the bold font-weight

Reply:
This is what it looks like when it is not bold, but it seems to be thinner than the font-weight in the prototype. Should I still make HomeTitle not bold?

## Not bold
![image](https://user-images.githubusercontent.com/51216746/140665258-59334b9e-0f5e-4952-b41f-490dfcac8017.png)

## Prototype
![image](https://user-images.githubusercontent.com/51216746/140665286-6487fec0-1bac-47d6-8b1f-54ad5bd8b673.png)

## Bold
![image](https://user-images.githubusercontent.com/51216746/140665302-fdfce40b-8579-45a8-b035-594a740a2cb3.png)

Reply:
Thanks for this. Let's keep it as is then